src/components/ScrollIndicator/ScrollIndicator.jsx

```diff
@@ -8,29 +10,74 @@ import './ScrollIndicator.scss';
 
 const getBlocksWith = createBemBlockBuilder(['indicatory']);
 
-export const ScrollIndicator = ({ sections }) => (
-  <>
-    {sections && (
-      <div className={getBlocksWith()}>
-        <div className={getBlocksWith('__box')}>
-          {sections.map((section) => (
-            <div key={section.id} className={getBlocksWith('__box-item')}>
-              <Link
-                className={getBlocksWith('__link')}
-                activeClass="active"
-                to={section.id}
-                spy={true}
-                smooth={true}
-                offset={-70}
-                duration={500}
-              >
-                {section.step && <span>{section.step}</span>}
-                <span className={getBlocksWith('__box-item-substring')}>{section.title}</span>
-              </Link>
+const FIRST_POSITION_OFFSET = 200;
+const OFFSET = 20;
+const HERO_HIGHT = 500;
```

Comment:
Should it be like that?
```suggestion
const HERO_HEIGHT = 500;
```